src/main/java/AlexLink/HomeWork/HW_Generics/Generics.java

```diff
@@ -0,0 +1,11 @@
+package AlexLink.HomeWork.HW_Generics;
+
+
+
+public class Generics<T> {
+    T[] array;
+
+    public T getArray(int number) {
```

Comment:
better name getArrayElement or getElement, because you get only one value

Reply:
fixed it